doc/plutus-core-spec/flat-serialisation.tex

```diff
@@ -412,6 +412,10 @@ \subsubsection{Terms}
   \Dterm(\bits{1001} \cdot s)  &= (s', \Kase{u}{l}) &&\quad \text{if } \Dterm(s) = (s', u) &&\ \text{and}\ \Dlist_{\mathsf{term}}(s') = (s'', l)
 \end{alignat*}
 
+\paragraph{NOTE.} The decoder $\Dterm$ should fail if we are decoding a program
```

Comment:
Strictly I should have term decoders indexed by version numbers, but I was lazy and decided to just fudge it.

Reply:
Seems fine for now